assets/app/views/create.html

```diff
@@ -3,13 +3,13 @@
     <div class="container">
       <ol class="breadcrumb">
         <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
-        <li class="active"><strong>Create</strong></li>
+        <li class="active"><strong>Add to Project</strong></li>
       </ol>
       <div class="row">
         <div class="col-md-12">
-          <h1>Create Using Your Code</h1>
+          <h1>Add Your Code</h1>
           <form name="from_source_form" class="form-inline gutter-bottom">
-            <p>Create your application from a Git source code repository.</p>
```

Comment:
I think I like the old description better... you're not just adding code, you're creating a whole lot of stuff around your code